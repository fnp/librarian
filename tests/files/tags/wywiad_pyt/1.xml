<wywiad_pyt><akap>blok akapitów pytania</akap></wywiad_pyt>

<wywiad_odp><akap>blok akapitów odpowiedzi</akap></wywiad_odp>
