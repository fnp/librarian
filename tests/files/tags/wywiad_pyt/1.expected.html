<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">blok akapitów pytania</p>
<a href="#f2" class="wl-num">2</a><p class="wl paragraph" id="f2">blok akapitów odpowiedzi</p>
</div>
